packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -136,6 +136,10 @@ export class ApmConfiguration {
       config.serverUrl = process.env.ELASTIC_APM_SERVER_URL;
     }
 
+    if (process.env.ELASTIC_APM_SECRET_TOKEN) {
```

Comment:
All these environemnt values are also picked up automatically by `apm-node-agent`.
However, we should be consistent, so I added this one here (`ELASTIC_APM_SERVER_URL` was passed in but `ELASTIC_APM_SECRET_TOKEN` was fetched by the agent)